LEAF_Request_Portal/sources/Form.php

```diff
@@ -3356,6 +3360,11 @@ public function query($inQuery)
                         FROM notes
                         WHERE recordID IN (' . $recordIDs . ')
                         AND deleted IS NULL
+                        UNION
```

Comment:
I haven't worked with a lot of UNION's before but just want to make sure this will be more efficient on resources vs separate querying.

Reply:
From my research doing a UNION is the same as calling 3 separate queries, essentially it is 3 queries run simultaneously and put together in one result.